client/i18n/locales/korean/motivation.json

```
@@ -0,0 +1,196 @@
+{
+  "compliments": [
+    "정말 자랑스러워요!",
+    "대단해요!",
+    "한 발자국 나아간 당신, 축하해요!",
+    "이 노력에 대한 보람은 무한할거예요.",
+    "당신은 꾸준히 나아가고 있어요!",
+    "당당하게 나아가는 모습이 멋져요!",
+    "바로 그거예요!",
+    "명중!",
+    "지화자!",
+    "무한 감탄중!",
+    "앵콜!",
+    "대.박.",
+    "무한 감탄중!",
+    "어마어마한 능력자군요!",
+    "얼쑤!",
+    "정상을 향해!",
+    "홈런입니다!",
+    "슛~ 골!",
+    "어벤져스도 감탄할 만한 실력!",
+    "기립 박수를 받으세요!",
+    "그 누구도 멈출 수 없다!",
+    "레벨 업!",
+    "풀파워 모드!",
+    "깔끔히 끝냈어요.",
+    "멋져요!",
+    "최고!",
+    "앵콜!",
+    "이 해피엔딩 영화의 주인공은 당신!",
+    "화이팅! 해야죠!",
+    ""
```

```suggestion
    "화이팅! 해야죠!"
```
If you leave an empty string in here there could be no compliment appearing

---

oops my bad, thanks @ilenia-magoni 😄